webrtc.html

```diff
@@ -11498,19 +11498,70 @@ <h3 id="mediastreamtrack-network-use">MediaStreamTrack</h3>
       <section>
         <h4>MediaTrackSupportedConstraints, MediaTrackCapabilities,
         MediaTrackConstraints and MediaTrackSettings</h4>
-        <p>The basics of <code>MediaTrackSupportedConstraints</code>,
-        <code>MediaTrackCapabilites</code>, <code>MediaTrackConstraints</code>
-        and <code>MediaTrackSettings</code> is outlined in [[!GETUSERMEDIA]].
-        However, the <code>MediaTrackSettings</code> for a
-        <code>MediaStreamTrack</code> sourced by an
-        <code><a>RTCPeerConnection</a></code> will only be populated with
-        members to the extent that data is supplied by means of the remote
-        <code><a>RTCSessionDescription</a></code> applied via
-        <code>setRemoteDescription</code> and the actual RTP data. This means
-        that certain members, such as <code>facingMode</code>,
-        <code>echoCancellation</code>, <code>latency</code>,
-        <code>deviceId</code> and <code>groupId</code>, will always be
-        missing.</p>
+        <p>The concept of constraints and constrainable properties, including
+        <code>MediaTrackConstraints</code>
+        (<code>MediaStreamTrack.getConstraints()</code>,
+        <code>MediaStreamTrack.applyConstraints()</code>), and
+        <code>MediaTrackSettings</code>
+        (<code>MediaStreamTrack.getSettings()</code>) are outlined in
+        [[!GETUSERMEDIA]]. However, the constrainable properties of tracks
+        sourced from an <code>RTCPeerConnection</code> are different than those
+        sourced by <code>getUserMedia()</code>. The constraints and settings
+        applicable in WebRTC are defined here. The settings of a track sourced
+        from an <code>RTCPeerConnection</code> represent the currently received
+        frames/samples after downsampling based on applied constraints, if
+        applied. <code>MediaStreamTrack.getCapabilities()</code> MUST always
+        return the empty set.</p>
+        <p>The following constrainable properties are defined to apply only to
```

Comment:
Grammar : "are defined to apply only" -> "are defined to apply to... ", and then add a sentence saying "The effect of all other constraints is not defined by this specification".